packages/figma-utils/src/variables/generate.ts

```diff
@@ -65,16 +76,24 @@ export const generateAsSCSS = (data: ParsedVariable, options?: BaseGenerateOptio
  *
  * @returns File content of the .json file
  */
-export const generateAsJSON = (data: ParsedVariable): string => {
-  const variables = structuredClone(data.variables);
+export const generateAsJSON = (data: ParsedVariable, dataDarkTheme?: ParsedVariable): string => {
+  const lightVars = structuredClone(data.variables);
+  const darkVars = structuredClone(dataDarkTheme?.variables ?? {});
 
-  // recursively resolve aliases to plain values since keys can not be referenced in a .json file
-  // like we could e.g. in a .css file
-  Object.keys(variables).forEach((name) => {
-    variables[name] = resolveValue(name, variables);
+  // Aliase auflösen
```

Comment:
```suggestion
  // recursively resolve aliases to plain values since keys can not be referenced in a .json file
  // like we could e.g. in a .css file
```